### PR TITLE
Added Press Bio

### PR DESCRIPTION
**Pull request by margarita-poole**
What I changed: 
-I added a content section wrapper with the press biography inside of it 
-I removed the links for the bio pdf and the comedy tour pdf

### CODE REVIEW

File: components/organisms/PressGallery/PressGallery.tsx [hunk 1]
```diff
@@ -32,46 +34,20 @@ const PressGallery = () => {
...
-              alt="YassComedyTourDescriptionThumbnail"
-              width={225}
-              height={300}
-            />
-            <span className="flex flex-col text-xl underline underline-offset-2 justify-center">
-              {'Comedy Tour Description PDF'}
-            </span>
-          </a>
+        <div className="flex flex-col mt-8 md:mt-0 md:ml-4">
+          <p className="text-4xl text-center">{`Press Bio`}</p>
```
Comment (Earle-Poole) [inline components/organisms/PressGallery/PressGallery.tsx]: `Press Bio` should probably be `Biography` here.

File: components/organisms/PressGallery/PressGallery.tsx [hunk 2]
```diff
@@ -32,46 +34,20 @@ const PressGallery = () => {
           </div>
         </div>
       </div>
-      {/* <h2 className={h2Styles}>{'Bio & Tour Flyer'}</h2>
-      <div
-        data-name="pdf section"
-        className="flex w-full justify-evenly my-10 mx-auto"
+      <ContentSectionWrapper
```
Comment (Earle-Poole) [inline components/organisms/PressGallery/PressGallery.tsx]: I think this should go before the "Banner Posters" section.

File: components/organisms/PressGallery/PressGallery.tsx (hunk 1 above)
Comment (margarita-poole) [reply components/organisms/PressGallery/PressGallery.tsx]: I renamed this one.

File: components/organisms/PressGallery/PressGallery.tsx (hunk 2 above)
Comment (margarita-poole) [reply components/organisms/PressGallery/PressGallery.tsx]: I moved this up but not sure if the page has the immediate wow factor anymore. Thoughts?

**Review by Earle-Poole — APPROVED** [review]
LGTM 👍

**Pull request merged**